m2m-transfers/LETTERSPECS/BANNO.M2MTRANSFERS.V1.CFG

```diff
@@ -97,11 +98,26 @@ XA: FALSE
 XT: SHARES
 *
 * ALLOW EDIT OF MANUALLY CREATED TRANSFERS
-* Line must begin with "AE:"
-* Should the program allow the member to edit manually created transfer
-* records?
+*  Line must begin with "AE:"
+*  Should the program allow the member to edit manually created transfer
+*  records?
 *  VALID VALUES: TRUE or FALSE
 *  DEFAULT: FALSE
 AE: FALSE
 *
+* ALLOW TRANSFERS TO CLUBS
+*  Line must begin with "ACL:"
+*  Should the program allow transfers to Club Shares (share code=3)?
+*  VALID VALUES: TRUE or FALSE
+*  DEFAULT: FALSE
+ACL: FALSE
+*
+* ALLOW TRANSFERS TO CERTIFICATES
+*  Line must begin with "ACE:"
+*  Should the program allow transfers to Club Shares (share code=3)?
```

Comment:
This line 117 would say Certificate Share, and the share code would be 2.
The rest of the config file changes look good.

Reply:
Corrected